packages/gafl-webapp-service/src/pages/contact/contact/contact.njk

```diff
@@ -85,22 +86,33 @@
      </div>
 {% endset -%}
 
-{% set noneHtml %}
-    {{ govukWarningText({
-      text: "Make a record of your licence number after payment",
-      iconFallbackText: "Warning",
-      classes: "govuk-!-margin-top-3"
-    }) }}
-    <p class="govuk-body-m">When you go fishing you must be able to show it to our enforcement officers if asked.</p>
+{% set postMessageHtml %}
+    <p class="govuk-body-m">If you want to fish before you receive the card you should make a note of the licence number after payment.</p>
+{% endset -%}
+
+{% set noneMessageHtml %}
+    <p class="govuk-body-m">We will show you the licence number on confirmation.</p>
 {% endset -%}
 
 {% block pageContent %}
-    <p class="govuk-body-m">We will normally only send you messages about the licence such as payment confirmation, licence details and renewal reminders.</p>
+    {% if data.isPhysical %}
+        <p class="govuk-body-m">You will receive the licence card in the post within 10 working days.
+        If you provide an email or mobile number we'll send you a confirmation message with the licence details.
+        You will also receive renewal reminders when your licence is expiring. </p>
+    {% else %}
+        {% if data.isJunior %}
+            <p class="govuk-body-m">We don't provide physical cards for junior licences.</p>
+            <p class="govuk-body-m">You will need to confirm your licence number if asked by an enforcement officer.</p>
+        {% else %}
+            <p class="govuk-body-m">We don't provide physical cards for 1 or 8 day licences licences.</p>
```

Comment:
Yes we do - but only for Salmon and Sea Trout 1 and 8 day licences

Reply:
I don't think thats right. I checked with both Ilia and the UCD team and they say no - the cards are for 365 day licences only. For a one day licence they would not get the card in time to fish. It makes no sense.

Reply:
Cards only sent for 365d